### PR TITLE
Proposed changes after reading mailing list

### PR DESCRIPTION
**Pull request by TaorLegrand**
Added Static Flags to required data that don't require polling.
Added DEFAULT_ONDELAY / OFFDELAY to load.on.delay / load.off.delay

### CODE REVIEW

File: drivers/legrand-hid.c [hunk 1]
```diff
@@ -119,31 +119,31 @@ static hid_info_t legrand_hid2nut[] = {
...
 
 	/* Output Data */
 	{ "output.voltage", 0, 0, "UPS.Output.Voltage", NULL, "%.0f", 0, NULL },
 	{ "output.voltage", 0, 0, "UPS.PowerConverter.Output.Voltage", NULL, "%.0f", 0, legrand_times10M_info },
 	{ "output.frequency", 0, 0, "UPS.Output.Frequency", NULL, "%.0f", 0, NULL },
 	{ "ups.load", 0, 0, "UPS.Output.PercentLoad", NULL, "%.0f", 0, NULL },
 	{ "ups.load", 0, 0, "UPS.OutletSystem.Outlet.PercentLoad", NULL, "%.0f", 0, NULL },
-	{ "ups.realpower", 0, 0, "UPS.Output.ConfigActivePower", NULL, "%.0f", 0, NULL },
-	{ "ups.realpower", 0, 0, "UPS.Flow.ConfigApparentPower", NULL, "%.0f", 0, NULL },
+	{ "ups.realpower", 0, 0, "UPS.Output.ConfigActivePower", NULL, "%.0f", HU_FLAG_STATIC, NULL },
```
Comment (zykh) [inline drivers/legrand-hid.c]: is it the nominal power? if so, it should be `ups.realpower.nominal` (also the one below).

File: drivers/legrand-hid.c [hunk 2]
```diff
@@ -119,31 +119,31 @@ static hid_info_t legrand_hid2nut[] = {
 	/* Input Data */
 	{ "input.voltage", 0, 0, "UPS.Input.Voltage", NULL, "%.0f", 0, legrand_times10_info },
 	{ "input.voltage", 0, 0, "UPS.PowerConverter.Input.Voltage", NULL, "%.0f", 0, legrand_times1M_info },
-	{ "input.transfer.high", 0, 0, "UPS.Input.HighVoltageTransfer", NULL, "%.0f", 0, NULL },
-	{ "input.transfer.high", 0, 0, "UPS.PowerConverter.Output.HighVoltageTransfer", NULL, "%.0f", 0, legrand_times10_info },
-	{ "input.transfer.low", 0, 0, "UPS.Input.LowVoltageTransfer", NULL, "%.0f", 0, NULL },
-	{ "input.transfer.low", 0, 0, "UPS.PowerConverter.Output.LowVoltageTransfer", NULL, "%.0f", 0, NULL },
-	{ "input.voltage.nominal", 0, 0, "UPS.Input.ConfigVoltage", NULL, "%.0f", 0, NULL },
-	{ "input.voltage.nominal", 0, 0, "UPS.Flow.ConfigVoltage", NULL, "%.0f", 0, NULL },
+	{ "input.transfer.high", 0, 0, "UPS.Input.HighVoltageTransfer", NULL, "%.0f", HU_FLAG_STATIC, NULL },
```
Comment (zykh) [inline drivers/legrand-hid.c:122]: so, the user is not allowed to change the value of the various `input.transfer.high`/`low` and `battery.charge.low`/`warning` items, right?

**Review by zykh — CHANGES_REQUESTED** [review]

File: drivers/legrand-hid.c (hunk 2 above)
Comment (TaorLegrand) [reply drivers/legrand-hid.c:122]: Right, it's not allowed

File: drivers/legrand-hid.c (hunk 1 above)
Comment (TaorLegrand) [reply drivers/legrand-hid.c]: you are right, it's ups.realpower.nominal instead of ups.realpower

**Review by zykh — APPROVED** [review]

**Pull request merged**